plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -180,19 +221,32 @@ public String getIncludePattern() {
         return includePattern;
     }
 
+    /**
+     * @see {@link #getIncludePattern()}
+     */
+    @DataBoundSetter
+    public void setIncludePattern(String includePattern) {
+        this.includePattern = includePattern;
+    }
+
     @Override
     public Action getProjectAction(final AbstractProject<?, ?> project) {
         return new FindBugsProjectAction(project);
     }
 
     @Override
-    public BuildResult perform(final AbstractBuild<?, ?> build, final PluginLogger logger) throws InterruptedException, IOException {
+    public BuildResult perform(final Run<?, ?> build, final FilePath workspace, final PluginLogger logger) throws InterruptedException, IOException {
```

Comment:
I think this is wrong - you have changed the behavior of the class as it no longer takes into account the thresholds (due to the parent no longer calling this.
Or I'm missing something as the [parent](https://github.com/jenkinsci/analysis-core-plugin/blob/master/src/main/java/hudson/plugins/analysis/core/HealthAwarePublisher.java#L148-L176)  hasn't had any changes in master and there is no SNAPSHOT update in the POM.